src/components/CatItem/styles.jsx

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+import { mobile } from "../../responsive";
+import { colors } from "../../themes";
+
+export const Container = styled.div`
+  border-radius: 2rem;
+  overflow: hidden;
+  flex: 1;
+  margin: 0.5rem;
+  height: 70vh;
+  position: relative;
+`;
+export const Image = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  ${mobile({ height: "30vh" })}
+`;
+export const TextHead = styled.div`
```

Comment:
like the use of ` for the effective styling of the components

Reply:
thanks